src/types/component.go

```diff
@@ -121,7 +122,7 @@ type ZarfChart struct {
 	RepoName    string              `json:"repoName,omitempty" jsonschema:"description=The name of a chart within a Helm repository (defaults to the Zarf name of the chart)"`
 	GitPath     string              `json:"gitPath,omitempty" jsonschema:"description=(git repo only) The sub directory to the chart within a git repo,example=charts/your-chart"`
 	LocalPath   string              `json:"localPath,omitempty" jsonschema:"description=The path to a local chart's folder or .tgz archive"`
-	Namespace   string              `json:"namespace" jsonschema:"description=The namespace to deploy the chart to"`
+	Namespace   string              `json:"namespace,omitempty" jsonschema:"description=The namespace to deploy the chart to"`
```

Comment:
Are there any known side effects to making the `.charts.namespace` field optional rather than required?

Reply:
We still have runtime validation after compose to ensure this is set https://github.com/defenseunicorns/zarf/blob/main/src/types/validate.go#L294-L296.

The issue with having it required is that it can be set by an import so vscode and `zarf dev lint` will incorrectly tell users their Zarf file is breaking the schema.